sematic/api/endpoints/resolutions.py

```diff
@@ -406,3 +407,40 @@ def _update_resolution_user(
     resolution.settings_env_vars[var_key] = user.api_key
 
     return resolution, True
+
+
+@sematic_api.route("/api/v1/resolutions/with_orphaned_jobs", methods=["GET"])
```

Comment:
This isn't an actual resource, and the naming suggests a smell.
Should this be a filter on the existing `resolutions` GET endpoint instead?

Reply:
I agree with this.
I think Josh mentioned the query logic would be difficult to unify between `/resolutions` and `/resolutions?orphaned_jobs=true`.
Maybe we can have an explicit branching in the endpoint code to accommodate both query logic. Nicer interface, uglier code.

Reply:
If we go that path we would also need to have some logic to say that you couldn't combine `orphaned_jobs=true` with other filters/group by logic (once we support searching on these APIs). We'd return an error if you tried to combine them. Are we ok with that?

Reply:
That sounds fine. We can think of a more generic filtering system if it ever becomes necessary in the future.